src/notebooks/outputs/tracebackFormatter.ts

```diff
@@ -48,9 +48,6 @@ export class NotebookTracebackFormatter implements ITracebackFormatter {
 
         const inputMatch = /^Input.*?\[.*32mIn\s+\[(\d+).*?0;36m(.*?)\n.*/.exec(traceFrame);
         if (inputMatch && inputMatch.length > 1) {
-            const executionCount = parseInt(inputMatch[1]);
-            // Don't assume the current cell is where the error is.
```

Comment:
This wasn't working if the execution count is the same in a previous cell. Not sure how in a notebook formatter the error would come from a different cell? Removed this as I don't think we actually need it.